ci/ReleaseTool/Common.cs

```diff
@@ -1,14 +1,83 @@
-using NLog;
+using Newtonsoft.Json.Linq;
 using System;
+using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Text;
 using System.Text.RegularExpressions;
 
 namespace ReleaseTool
 {
     internal static class Common
     {
-        public const string RepoUrlTemplate = "git@github.com:{0}/{1}.git";
+        private const string RepoUrlTemplate = "git@github.com:{0}/{1}.git";
+
+        // Names of the version files that live in the UnrealEngine repository.
+        private const string UnrealGDKVersionFile = "UnrealGDKVersion.txt";
+        private const string UnrealGDKExampleProjectVersionFile = "UnrealGDKExampleProjectVersion.txt";
+        private const string UnrealGDKTestGymVersionFile = "Engine/Plugins/UnrealGDK/UnrealGDKTestGymVersion.txt";
```

Comment:
```suggestion
        private const string UnrealGDKTestGymVersionFile = "UnrealGDKTestGymVersion.txt";
```
If it lives in the GDK repo, as we are in the root of the repo at that point, no?